juriscraper/pacer/appellate_docket.py

```diff
@@ -686,17 +695,19 @@ def _get_tail_by_regex(self, regex, cast_to_date=False, node=None):
                 return convert_date_string(tail)
             return tail
 
-
-if __name__ == "__main__":
+def _main():
     if len(sys.argv) != 2:
-        print("Usage: python -m juriscraper.pacer.appellate_docket filepath")
-        print("Please provide a path to an HTML file to parse.")
+        print "Usage: python -m juriscraper.pacer.appellate_docket filepath"
```

Comment:
This came up in linting, I presume? I'd leave these because they're prep for Py3 conversion.